src/standard-stores/index.ts

```diff
@@ -10,29 +10,32 @@ import {
   writable as vanillaWritable,
 } from 'svelte/store';
 import { anyReloadable, loadAll, reloadAll } from '../utils';
-import type { Loadable, Stores, StoresValues } from '../async-stores/types';
-
-const loadDependencies = <S extends Stores, T>(
+import type {
+  Loadable,
+  Stores,
+  StoresValues,
+  VisitedMap,
+} from '../async-stores/types';
+
+const loadDependencies = async <S extends Stores, T>(
```

Comment:
changes this to be an async function rather than it being an async function generator. This made it easier to work with.